README.md

```diff
@@ -56,6 +58,10 @@ zones:
 
 `class` is a special key that tells OctoDNS what python class should be loaded. Any other keys will be passed as configuration values to that provider. In general any sensitive or frequently rotated values should come from environmental variables. When OctoDNS sees a value that starts with `env/` it will look for that value in the process's environment and pass the result along.
 
+`enforce_order` is a special key that tells OctoDNS to enforce sorting order on the yaml config. By default it is set to True.
+
+If you don't want to set same TTL each time for your records, you can optionally use `default_ttl` key - it will be used in records when not specified in the data. By default it is set to 3600 seconds.
+
```

Comment:
`enforce_order` and `default_ttl` are both specific to `YamlProvider` and not part of the stuff being documented here (the manager config file.) It seems fine/reasonable to include them in the example up above where `config` is used, but documenting the provider stuff here is probably a bit out of scope. `YamlProvider` does have a class docstring that includes info about them. Not opposed to having separate markdown documentation for the providers, but don't think it belongs here.

Reply:
Ok, I'll remove `enforce_order` and `default_ttl` description for now. Maybe it's really better to create separate doc for `YamlProvider` and others.